### PR TITLE
#290 Adding spaces to make tree look like the flat excel spreadsheet

### PR DESCRIPTION
**Pull request by mmilitaru**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/pulls) open
- [x] I have verified that I am following the CDP4-IME [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/CDP4-IME-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
#290 adding indentation to the spreadsheet to simulate treeview like presentation.

### CODE REVIEW

**Comment by mmilitaru** [comment]
#290 A test case has been added

File: CDP4ParameterSheetGenerator/RowModels/OptionSheet/NestedElementExcelRow.cs [hunk 1]
```diff
@@ -91,6 +103,9 @@ private void ProcessNestedParameters(IEnumerable<NestedParameter> nestedParamete
                     }
 
                     var nestedParameterExcelRow = new NestedParameterExcelRow(nestedParameter);
+                    nestedParameterExcelRow.Level = this.Level + 1;
```
Comment (lxatrhea) [inline CDP4ParameterSheetGenerator/RowModels/OptionSheet/NestedElementExcelRow.cs]: This works ok, but shouldn't the NestedParameterExcelRow class itself be responsible for adding spaces to its name?

File: CDP4ParameterSheetGenerator.Tests/RowModels/NestedParameterExcelRowTestFixture.cs [hunk 2]
```diff
@@ -123,6 +123,50 @@ public void TearDown()
...
+            parameter.ValueSet.Add(parameterValueSet);
+            this.satellite.Parameter.Add(parameter);
+
+            var nestedElementTreeGenerator = new NestedElementTreeGenerator();
+            var nestedElements = nestedElementTreeGenerator.Generate(this.option, this.systemEngineering);
+
+            var elementList = nestedElements.ToList();
+            var rootnode = elementList.Single(ne => ne.ElementUsage.Count > 0);
+
+            if (rootnode != null)
```
[the remark this answers is not in the document]
Comment (alexatrhea) [reply CDP4ParameterSheetGenerator.Tests/RowModels/NestedParameterExcelRowTestFixture.cs]: I agree

File: CDP4ParameterSheetGenerator/RowModels/OptionSheet/NestedElementExcelRow.cs (hunk 1 above)
Comment (alexatrhea) [reply CDP4ParameterSheetGenerator/RowModels/OptionSheet/NestedElementExcelRow.cs]: Well, I think here this is ok. NestedParameterExcelRow in principle should never exist outside of a NestedElement and its level should be directly tied to the said element.

**Pull request merged**